test/browser/spec/ol/renderer/vector.test.js

```diff
@@ -50,14 +50,14 @@ describe('ol.renderer.vector', function () {
 
         expect(iconStyleLoadSpy.calledOnce).to.be.ok();
         listeners = iconStyle.iconImage_.listeners_['change'];
-        expect(listeners.length).to.eql(1);
+        expect(listeners.length).to.eql(2);
```

Comment:
Where does the extra listener come from? Are we doing unnecessary extra work somewhere?

Reply:
I'm using `listenImageChange` - instead to using `getImage()` then listening for its load event, which was causing  `initializeImage` to be called sooner than necessary and incompatible with Node tests.